analytics/file_download_per_email_domain.sql

```diff
@@ -0,0 +1,30 @@
+-- This was a small query for https://sagebionetworks.jira.com/browse/SNOW-135
+-- The query is to extract email domains and provide a count of unique downloads per email domain
+USE ROLE DATA_ANALYTICS;
+USE WAREHOUSE COMPUTE_XSMALL;
+USE DATABASE SYNAPSE_DATA_WAREHOUSE;
+
+
+WITH DEDUP_FILEHANDLE AS (
+    SELECT DISTINCT
+        USER_ID,
+        FILE_HANDLE_ID AS FD_FILE_HANDLE_ID,
+        RECORD_DATE,
+        PROJECT_ID
+    FROM
+        SYNAPSE_DATA_WAREHOUSE.SYNAPSE.FILEDOWNLOAD
```

Comment:
Nit: just a note here, you can explore the "association_object_type" column.  That will tell you "what" the downloads are associated it. You'll see something like this:
<img width="853" alt="image" src="https://github.com/user-attachments/assets/5b206af9-92c0-485f-b422-28bc58368265">

One thing we need to figure out at Sage is _what_ we want to consider as a "download"